keepalived/check/check_misc.c

```diff
@@ -71,6 +71,26 @@ dump_misc_check(void *data)
 	log_message(LOG_INFO, "   insecure = %s", misck_checker->insecure ? "Yes" : "No");
 }
 
+static int
+compare_misc_check(void *a, void *b)
+{
+	misc_checker_t *old = CHECKER_DATA(a);
+	misc_checker_t *new = CHECKER_DATA(b);
+
+	if (strcmp(old->path, new->path) != 0)
+		goto err;
+	if (old->timeout != new->timeout)
+		goto err;
+	if (old->dynamic != new->dynamic)
+		goto err;
```

Comment:
Similar to retry above, I don't think different values of timeout or dynamic make this a different checker